ui/src/app/views/rename-modal/component.html

```diff
@@ -0,0 +1,17 @@
+<clr-modal [(clrModalOpen)]="renaming" [clrModalSize]="'md'">
+    <div class="modal-title">
+        <!--The ngIf is to avoid null pointers before the modal is opened.-->
+        <div *ngIf="renaming" class="modal-title-text">Please enter the new name for {{resource.name}}</div>
+    </div>
+    <div class="modal-body">
+        <div class="form-section">
+            <label>New Name: </label>
+            <input id="new-notebook-name" type="text" class="input"
```

Comment:
Nit: I would not call this `new-notebook-name` because its a generalized component. Maybe `resource-name`?

Reply:
Already changed to new-name (not pushed yet - as I mentioned, not quite ready for reviews yet!)